lib/tapioca/helpers/rbi_files_helper.rb

```diff
@@ -48,13 +48,16 @@ def index_rbis(index, kind, dir)
     def duplicated_nodes_from_index(index, shim_rbi_dir:, todo_rbi_file:)
       duplicates = {}
       say("Looking for duplicates... ")
-      index.keys.each do |key|
-        nodes = index[key]
-        next unless shims_or_todos_have_duplicates?(nodes, shim_rbi_dir: shim_rbi_dir, todo_rbi_file: todo_rbi_file)
+      time = Benchmark.measure do
```

Comment:
To avoid repeating the pattern `result = Benchmark.measure {}; result.real`, you can use `Benchmark.realtime` which just returns the number straight away.

Reply:
Even better 👍